.hygen/generate/document-resource/database/model.ejs.t

```diff
@@ -1,24 +1,24 @@
 ---
-to: src/database/models/<%= h.inflection.transform(name, ['underscore', 'dasherize']) %>.model.ts
```

Comment:
the name of model if CarCategory it's file name should be car-category.model.ts not carCategory.model.ts